src/index.js

```diff
@@ -17,7 +18,7 @@ const bootstrap = () => {
         ReactDOM.render(<Demo/>, document.getElementById('root'));
```

Comment:
Can we take this HomePage inside <Demo/>, so that all stay inside demo, and <App/> will have final integrated app, and we can use `npm run start:demo` for demo,  `npm run start `for final App and `npm run start:kai-ui` for KaiUI our parent components library

Reply:
hey @shishirarora3 actually when I run the command 'npm run start:demo' error is coming that REACT_APP_DEMO not defined. I think environment variable is not automatically getting imported to process.env in windows. Command that is mentioned in script file does get executed but still is not recognizing the env variable.